Pcap++/src/PcapLiveDevice.cpp

```diff
@@ -44,6 +44,7 @@
 #define LIBPCAP_OPEN_LIVE_TIMEOUT -1
 #endif
 
+static const char *NFLOG_IFACE = "nflog";
```

Comment:
I'm not familiar with nflog... Is the interface name must be `nflog`? Or is it configurable?

Reply:
Yes. The interface name must be `nflog:<NFLOG_GROUP>` or simply `nflog`, in which case NFLOG_GROUP is defaulted to 0.

Reply:
Also, that's how `libpcap` interacts with this interface.